sormas-api/src/main/java/de/symeda/sormas/api/caze/CaseCriteria.java

```diff
@@ -560,4 +562,13 @@ public Boolean getIncludeCasesFromOtherJurisdictions() {
 	public void setIncludeCasesFromOtherJurisdictions(Boolean includeCasesFromOtherJurisdictions) {
 		this.includeCasesFromOtherJurisdictions = includeCasesFromOtherJurisdictions;
 	}
+
+	public ContactReferenceDto getContact() {
+		return contact;
+	}
+
+	public CaseCriteria contact(ContactReferenceDto contact) {
```

Comment:
Did I miss something or is this never actually used? If so, it should be removed